examples/advanced/xgboost/fedxgb_secure/README.md

```diff
@@ -14,6 +14,9 @@ To be able to run all the examples, please install the requirements first.
 ```
 pip install -r requirements.txt
 ```
+
+If you want to run the secure vertical XGBoost with libnvflare.so plugin, you will need to install the [IPCL](https://github.com/intel/pailliercryptolib_python) library (by `git clone` and `pip install -e .`).
+
```

Comment:
I think we are providing mixed and broken info for plugin, let's have a discussion